src/Blocks/continuous.jl

```diff
@@ -20,7 +20,7 @@ Initial value of integrator state ``x`` can be set with `x`
 @mtkmodel Integrator begin
     @extend u, y = siso = SISO()
     @variables begin
-        x(t) = 0.0, [description = "State of Integrator"]
+        x(t), [guess = 0.0, description = "State of Integrator"]
```

Comment:
Integrators always start at zero though?

Reply:
oh, I wasn't sure about this one, I'll revert this.